third_party/3/waitress/adjustments.pyi

```diff
@@ -0,0 +1,60 @@
+from .compat import HAS_IPV6, PY2, WIN, string_types
+from .proxy_headers import PROXY_HEADERS
+from socket import SocketType
+from typing import Any, Mapping, Sequence, Optional, Set, Tuple, Union
+
+truthy: frozenset
+KNOWN_PROXY_HEADERS: frozenset
+
+def asbool(s: Optional[Union[bool, str, int]]) -> bool: ...
+def asoctal(s: str) -> int: ...
+def aslist_cronly(value: str) -> Sequence[str]: ...
+def aslist(value: str) -> Sequence[str]: ...
```

Comment:
The general rule is to use protocols like `Sequence` for argument types and concrete types, `List` in this case, for return types.